src/disco/gui/fd_gui.c

```diff
@@ -1284,6 +1284,9 @@ fd_gui_handle_completed_slot( fd_gui_t * gui,
     slot->compute_units   = compute_units;
   }
 
+  fd_gui_printf_slot( gui, _slot );
```

Comment:
I think the not publishing here is by design, since the slot will get published when it becomes part of the active fork chain. Otherwise every slot gets published twice.

Reply:
I see that makes sense